src/shapes/Group.ts

```diff
@@ -173,7 +173,7 @@ export class Group extends createCollectionMixin(
     objectsRelativeToGroup?: boolean
   ) {
     super();
-    this._objects = objects;
+    this._objects = objects.slice(); // Avoid unwanted mutations of Collection to affect the caller
```

Comment:
```suggestion
    this._objects = objects.slice(); // Avoid mutations
```

Reply:
I think it should be mentioned that the mutations come from Collection as it's not evident since the constructor is not mutating it

Reply:
ok